src/colorizer/shaders/colorize_RGBA8U.frag

```diff
@@ -91,6 +92,12 @@ vec4 getColorRamp(float val) {
   return texture(colorRamp, vec2(adjustedVal, 0.5));
 }
 
+vec4 getCategoricalColor(float featureValue) {
+  float width = float(textureSize(colorRamp, 0).x);
+  float modValue = mod(featureValue, width);
+  return getColorRamp(modValue / (width - 1.0));
+}
```

Comment:
All categorical color ramps are now repeated! The feature value is cast to an integer and then modded over the width of the color ramp texture.